docs/sdks/server-sdks/dart/quickstart.mdx

```diff
@@ -0,0 +1,246 @@
+---
+title: Quickstart
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+The Eppo Dart SDK enables feature flags and experiments in your Dart applications with only a few lines of code.
+
+The SDK handles all the complexity of feature flag evaluation and experiment assignment locally in your application. This guide will walk you through installing the SDK and implementing your first feature flag, experiment, and contextual bandit.
+
+## Installation
+
+First, install the SDK into your current project by adding it to your `pubspec.yaml` file:
```

Comment:
Can add the dep to the pubspec.yaml file with a command

`dart pub add eppo_sdk` or `flutter pub add eppo_sdk`